docs/src/getting_started/core.md

```diff
@@ -215,16 +215,39 @@ The `build.rs` file should now look like this:
 ```
 
 ````admonish tip
- You may also need to register any nested enum types (due to a current limitation with the reflection library, see <https://github.com/zefchain/serde-reflection/tree/main/serde-reflection#supported-features>).
-Here is an example of this from the [`build.rs`](https://github.com/redbadger/crux/blob/master/examples/notes/shared_types/build.rs) file in the `shared_types` crate of the [notes example](https://github.com/redbadger/crux/tree/master/examples/notes):
+Due to a current limitation with the reflection library,
+you may need to manually register nested enum types in your `build.rs` file.
+(see <https://github.com/zefchain/serde-reflection/tree/main/serde-reflection#supported-features>)
+
+
+*Note, you don't have to do this for the latest versions of the `crux_http`, `crux_kv` or `crux_time`
+capabilities, which now do this registration for you.*
```

Comment:
Do we need instructions to import the capabilities in `shared_types` with the `typegen` feature enabled?

Reply:
oh yeah

Reply:
https://github.com/redbadger/crux/pull/264/commits/ac1362a25974cff6693b6538b0dbd8e0e068de40